lms/djangoapps/ccx/migrations/0002_convert_memberships.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        "Convert CCX Memberships to Course Enrollments."
+        from ccx_keys.locator import CCXLocator
+        for membership in orm['ccx.CcxMembership'].objects.all():
+            #ccx = orm['ccx.CustomCourseForEdx'].objects.get(pk=membership.ccx)
+            ccx = membership.ccx
```

Comment:
Ack, minor other point -- can we do a `select_related` for ccx, so we're not doing a query on every loop iteration?